src/combinatorics/simple_graph/order.lean

```diff
@@ -0,0 +1,79 @@
+/-
+Copyright (c) 2021 Eric. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Eric Rodriguez
+-/
+import combinatorics.simple_graph.subgraph
+
+/-!
+# Order relation on simple graphs
+
+This file gives `simple_graph` a `bounded_lattice` instance under the `is_subgraph` relation, and
+proves a couple lemmata about how this relation interacts with `simple_graph.subgraph.spanning_coe`.
```

Comment:
Maybe should mention that this is the lattice of spanning subgraphs of the complete graph?